assets/index.css

```diff
@@ -19,17 +19,19 @@ main>*:not(:first-child) {
   background-color: var(--lawnchair-background-color);
   background-repeat: no-repeat;
   background-size: cover;
-  min-height: 90vh;
+  min-height: 100vh;
```

Comment:
The reason this isn't full width is to show that users can scroll the page. (Sometimes, scrollbars aren't enough to show the page is scrollable. Also mobile doesn't show them)